resources/views/bread/edit-add.blade.php

```diff
@@ -1,15 +1,20 @@
+@php
+    $edit = !is_null($dataTypeContent->getKey());
+    $add  = is_null($dataTypeContent->getKey());
```

Comment:
Do you need the `$add`? I don't see it used anywhere

Reply:
Not really. Just added it for consistency. Will remove it